public/404.html

```diff
@@ -1,67 +1,85 @@
 <!DOCTYPE html>
-<html>
+<html class="govuk-template">
 <head>
-  <title>The page you were looking for doesn't exist (404)</title>
-  <meta name="viewport" content="width=device-width,initial-scale=1">
-  <style>
-  .rails-default-error-page {
-    background-color: #EFEFEF;
-    color: #2E2F30;
-    text-align: center;
-    font-family: arial, sans-serif;
-    margin: 0;
-  }
-
-  .rails-default-error-page div.dialog {
-    width: 95%;
-    max-width: 33em;
-    margin: 4em auto 0;
-  }
+  <meta charset="utf-8" />
+  <title>Page not found – GOV.UK</title>
```

Comment:
Should include the following for better browser (mainly mobile) support.

```html
  <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
  <meta name="theme-color" content="#0b0c0c">

  <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Reply:
Added!